src/spec/orderexport.spec.coffee

```diff
@@ -97,6 +97,47 @@ describe 'OrderExport', ->
         done()
       .catch(done)
 
+  it '#runCSVAndStreamToFile should accept and filter by where parameter if present', (done) ->
+    callback = () -> Promise.resolve()
+    @orderExport._exportOptions.where = 'customerId="2878df1a-ea36-4434-a473-a8adcf1fc385"'
+    @orderExport._getCSVOrderTemplate = () ->
```

Comment:
We should not replace methods like this, better if we stub and replace the method so it can be restored.

Not too much of an issue because we are instantiating a new orderExport object every time.